modules/460-log-shipper/hooks/internal/vrl/cleanup.go

```diff
@@ -31,4 +31,8 @@ if exists(.kubernetes) {
 if exists(.file) {
     del(.file)
 }
+if exists(.node_labels."node.deckhouse.io/group") {
+	.node_group = (.node_labels."node.deckhouse.io/group")
+	del(.node_labels)
```

Comment:
```suggestion
}
del(.node_labels)
```

Reply:
The `node_labels` object should always be deleted despite having the `node.deckhouse.io/group` annotation.